### DISCUSSION TITLE
Building and using signatures in Futurenet/Local Networks for the advanced auth + Script for building signatures

### DISCUSSION THREAD

**Discussion by heytdep** [stellar/sorobanathon#39] [General]
Hey we've written a write-up + contract about using the advanced auth with secure/insecure signatures in the Futurenet/Local Network. The write-up also makes use of a script I (@heytdep) built to generate signatures for a given stellar keypair.

The write-up is in the readme of the repo: https://github.com/Xycloo/soroban-cli-futurenet-advanced-auth
You can find the script to build signatures here: https://github.com/Xycloo/soroban-build-signatures

**Comment by achallagundla** [comment]
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1250 XLM!
I just need to confirm your Public Key, sending you a message on Discord for that!
